### ISSUE TITLE
Do not check in messages file.

### ISSUE THREAD

**Issue by jordwalke** [facebook/reason#719]
The messages file causes all kinds of rebase issues. Are we able to not check it in, and only have it be generated at build/publish time?

**Comment by jordwalke** [comment]
cc @yunxing

**Comment by yunxing** [comment]
Unfortunately we can't get it to be generated as the error messages there are hand-written. Do you have a specific commit that has rebase issue that I can take a look at?

**Comment by anmonteiro** [comment]
No longer relevant after #2480
